Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  module_function
+
+  def bump_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `bump` [<options>] [<formula>]
+
+        Display out-of-date brew formulae and the latest version available.
+        Also displays whether a pull request has been opened with the URL.
+      EOS
+      flag "--limit=",
+           description: "Limit number of package results returned."
+      switch :verbose
+      switch :debug
+    end
+  end
+
+  def bump
+    args = bump_args.parse
+
+    requested_formulae = !args.formulae.empty? ? args.formulae.map(&:name) : nil
+
+    requested_limit = args.limit.to_i if args.limit.present?
+
+    repology_data = if requested_formulae
+      response = {}
+      requested_formulae.each do |formula|
+        raise FormulaUnavailableError, formula unless validate_formula(formula)
+
+        package_data = Repology.single_package_query(formula)
+        response[package_data.keys.first] = package_data.values.first if package_data
+      end
+
+      response
+    else
+      Repology.parse_api_response
```

Comment:
This seems like it should take the `requested_limit` parameter because otherwise it can query e.g. three pages of results (which is a bit slow).